doc/scoring_epochs.json

```diff
@@ -101,6 +101,10 @@
 					"start_time": 1658992027,
 					"start_time_comment": "Sat July 28 07:07:07 2022 +0000 commit TBA, timestamp may change"
 				},
+				"VRSC": {
+					"start_time": 1668898800,
```

Comment:
Not very significant, but I guess start time should be `1668902282`, as it [mentioned](https://github.com/VerusCoin/VerusCoin/commit/e4bdee1bdae802d3dd97a34a89db2acaa1491e16#diff-b532d5f4f7f151613d9551812182b064a77f762d49a359ad0fcba2a3ff5d5a96R414) in VRSC sources.

Reply:
technically scoring should start 24hr after update announcement, so this value is rarely precise initially, and eventually retroactively refined during final scoring calculations